virttest/qemu_devices/qcontainer.py

```diff
@@ -93,34 +96,18 @@ def get_qmp_cmds(qemu_binary, workaround_qemu_qmp_crash=False):
                 return cmds
 
         self.__state = -1    # -1 synchronized, 0 synchronized after hotplug
+        self.__machine_type = machine_type
```

Comment:
I'd use the term `__base_machine_type` or `__early_machine_type` as later in the code the machine_type could be different/changed/....